op-service/sources/batching/tx_call.go

```diff
@@ -0,0 +1,67 @@
+package batching
+
+import (
+	"github.com/ethereum-optimism/optimism/op-service/sources/batching/rpcblock"
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+type TxCall struct {
+	Abi    *abi.ABI
+	TxHash common.Hash
+	Method string
+}
+
+func NewTxCall(abi *abi.ABI, txhash common.Hash, method string) *TxCall {
```

Comment:
I feel `TxCall` is quite confusing - should it be `TxGetByHash` or something else?

Reply:
done